oak-segment-azure/src/test/resources/logback-test.xml

```diff
@@ -21,11 +21,19 @@
             <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
         </encoder>
     </appender>
-    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
```

Comment:
Is this related to switching to testcontainers?

Reply:
logs were getting printed on console, it is always better to have a log file, hence added the same.

Reply:
Then by all means open a separate ticket and create a second PR.

A single PR should address a single problem only (if possible).

Reply:
@reschke I cannot create a ticket in Oak, if you can create one please do that, I will raise a PR for same under that ticket.

Reply:
-> https://selfserve.apache.org/jira-account.html

Reply:
I have raised a request for jira access on the portal that you shared. I guess this will take some time, can we keep this change in this PR? Will keep in mind that a single PR should be related to a single jira.

Reply:
No, please move this into a separate PR.

Reply:
reverted back this change